docs/build_jpb.py

```diff
@@ -0,0 +1,139 @@
+import ast
```

Comment:
What does than name of the file stand for? `docs/build_jpb.py` - typo in job?

Reply:
build **J**u**P**yter**B**ook. Can rename to build_site_structure or similar